vercel.json

```diff
@@ -10,397 +10,5 @@
       ]
     }
   ],
-  "cleanUrls": true,
-  "redirects": [
```

Comment:
Are these all duplicates with the Astro redirects? We might need to merge this list into the Astro redirects file and clean up duplicate entries.

Reply:
Yes, and we shouldn't be using vercel redirections at all with astro 2.0 https://vercel.com/docs/frameworks/astro#can-i-use-vercel.json-to-create-rewrites-with-astro